### PR TITLE
fix: cannot automatically switch to available workspace after workspace is deleted

### PR DESCRIPTION
**Pull request by jzhangdev**

### CODE REVIEW

File: components/chat/chat-hooks/use-scroll.tsx
Comment (fkesheh) [inline components/chat/chat-hooks/use-scroll.tsx:1]: I would not do that in the same PR. Even if that is correct, which I'm not sure it's aligned with the prettier specs, it confuses the reviewer as he needs to go to other files. It also may pollute git blame. This in IMO

File: db/storage/files.ts
Comment (fkesheh) [inline db/storage/files.ts:1]: I would not do that in the same PR. Even if that is correct, which I'm not sure it's aligned with the prettier specs, it confuses the reviewer as he needs to go to other files. It also may pollute git blame. This in IMO

**Review by fkesheh — COMMENTED** [review]
It looks good to me, but it has minor format changes on two other files that are not directly related to this PR

File: components/chat/chat-hooks/use-scroll.tsx
Comment (jzhangdev) [reply components/chat/chat-hooks/use-scroll.tsx:1]: Thanks for the review, I've removed the extraneous changes.

**Pull request merged**